app/v2/htlc/internal/types/msgs.go

```diff
@@ -33,9 +33,9 @@ var _ sdk.Msg = &MsgRefundHTLC{}
 // MsgCreateHTLC represents a msg for creating an HTLC
 type MsgCreateHTLC struct {
 	Sender               sdk.AccAddress `json:"sender"`                  // the initiator address
-	Receiver             sdk.AccAddress `json:"receiver"`                // the recipient address
-	ReceiverOnOtherChain []byte         `json:"receiver_on_other_chain"` // the recipient address on other chain
-	Amount               sdk.Coin       `json:"amount"`                  // the amount to be transferred
+	To                   sdk.AccAddress `json:"to"`                      // the destination address
+	ReceiverOnOtherChain string         `json:"receiver_on_other_chain"` // the swap counter party address on the other chain
```

Comment:
The field refers to the address on the other chain on the behalf of the sender instead of the counterparty.